v4_abacus.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
     spec.name                     = 'v4_abacus'
-    spec.version                  = '1.13.6'
+    spec.version                  = '1.13.7'
```

Comment:
since we're removing properties from state this is technically a breaking change, right?

Reply:
No, it doesn't change the state exposed to the client.  We only removed it from the internal state.